### PR TITLE
More keyboard types

### PR DESCRIPTION
**Pull request by Polytoo**
Work out some more keyboard types

### CODE REVIEW

File: nx/include/switch/applets/swkbd.h [hunk 1]
```diff
@@ -18,9 +18,15 @@ typedef enum {
...
-    SwkbdType_Normal = 0,  ///< Normal keyboard.
-    SwkbdType_NumPad = 1,  ///< Number pad. The buttons at the bottom left/right are only available when they're set by \ref swkbdConfigSetLeftOptionalSymbolKey / \ref swkbdConfigSetRightOptionalSymbolKey.
-    SwkbdType_QWERTY = 2,  ///< QWERTY (and variants) keyboard only.
+    SwkbdType_Normal    = 0,  ///< Normal keyboard.
+    SwkbdType_NumPad    = 1,  ///< Number pad. The buttons at the bottom left/right are only available when they're set by \ref swkbdConfigSetLeftOptionalSymbolKey / \ref swkbdConfigSetRightOptionalSymbolKey.
+    SwkbdType_QWERTY    = 2,  ///< QWERTY (and variants) keyboard only.
+    SwkbdType_Unkown    = 3,  ///< The same as SwkbdType_Normal keyboard.
+    SwkbdType_Latin     = 4,  ///< Latin like languages keyboard only.
+    SwkbdType_CHS       = 5,  ///< Chinese Simplifid keyboard only.
+    SwkbdType_CHT       = 6,  ///< Chinese Simplifid keyboard only.
```
Comment (masagrator) [inline nx/include/switch/applets/swkbd.h]: Lines 26 & 27 need fixing comments.

Comment (Polytoo) [reply nx/include/switch/applets/swkbd.h]: Oh, sorry about that.

File: nx/include/switch/applets/swkbd.h [hunk 2]
```diff
@@ -18,9 +18,15 @@ typedef enum {
...
 typedef enum {
-    SwkbdType_Normal = 0,  ///< Normal keyboard.
-    SwkbdType_NumPad = 1,  ///< Number pad. The buttons at the bottom left/right are only available when they're set by \ref swkbdConfigSetLeftOptionalSymbolKey / \ref swkbdConfigSetRightOptionalSymbolKey.
-    SwkbdType_QWERTY = 2,  ///< QWERTY (and variants) keyboard only.
+    SwkbdType_Normal    = 0,  ///< Normal keyboard.
+    SwkbdType_NumPad    = 1,  ///< Number pad. The buttons at the bottom left/right are only available when they're set by \ref swkbdConfigSetLeftOptionalSymbolKey / \ref swkbdConfigSetRightOptionalSymbolKey.
+    SwkbdType_QWERTY    = 2,  ///< QWERTY (and variants) keyboard only.
+    SwkbdType_Unkown    = 3,  ///< The same as SwkbdType_Normal keyboard.
+    SwkbdType_Latin     = 4,  ///< Latin like languages keyboard only.
+    SwkbdType_CHS       = 5,  ///< Chinese Simplifid keyboard only.
```
Comment (masagrator) [inline nx/include/switch/applets/swkbd.h]: Still needs fix -> Simplified

File: nx/include/switch/applets/swkbd.h [hunk 3]
```diff
@@ -18,9 +18,15 @@ typedef enum {
 
 /// Type of keyboard.
 typedef enum {
-    SwkbdType_Normal = 0,  ///< Normal keyboard.
-    SwkbdType_NumPad = 1,  ///< Number pad. The buttons at the bottom left/right are only available when they're set by \ref swkbdConfigSetLeftOptionalSymbolKey / \ref swkbdConfigSetRightOptionalSymbolKey.
-    SwkbdType_QWERTY = 2,  ///< QWERTY (and variants) keyboard only.
+    SwkbdType_Normal    = 0,  ///< Normal keyboard.
+    SwkbdType_NumPad    = 1,  ///< Number pad. The buttons at the bottom left/right are only available when they're set by \ref swkbdConfigSetLeftOptionalSymbolKey / \ref swkbdConfigSetRightOptionalSymbolKey.
+    SwkbdType_QWERTY    = 2,  ///< QWERTY (and variants) keyboard only.
+    SwkbdType_Unkown    = 3,  ///< The same as SwkbdType_Normal keyboard.
```
Comment (masagrator) [inline nx/include/switch/applets/swkbd.h]: Unknown

Comment (Polytoo) [reply nx/include/switch/applets/swkbd.h]: I am so sorry for making so many spelling mistakes.
Thank you for your time, man.

File: nx/include/switch/applets/swkbd.h [hunk 4]
```diff
@@ -18,9 +18,15 @@ typedef enum {
 
 /// Type of keyboard.
 typedef enum {
-    SwkbdType_Normal = 0,  ///< Normal keyboard.
-    SwkbdType_NumPad = 1,  ///< Number pad. The buttons at the bottom left/right are only available when they're set by \ref swkbdConfigSetLeftOptionalSymbolKey / \ref swkbdConfigSetRightOptionalSymbolKey.
-    SwkbdType_QWERTY = 2,  ///< QWERTY (and variants) keyboard only.
+    SwkbdType_Normal    = 0,  ///< Normal keyboard.
+    SwkbdType_NumPad    = 1,  ///< Number pad. The buttons at the bottom left/right are only available when they're set by \ref swkbdConfigSetLeftOptionalSymbolKey / \ref swkbdConfigSetRightOptionalSymbolKey.
+    SwkbdType_QWERTY    = 2,  ///< QWERTY (and variants) keyboard only.
+    SwkbdType_Unknown   = 3,  ///< The same as SwkbdType_Normal keyboard.
```
Comment (yellows8) [inline nx/include/switch/applets/swkbd.h]: SwkbdType_Unknown3

File: nx/include/switch/applets/swkbd.h [hunk 5]
```diff
@@ -18,9 +18,15 @@ typedef enum {
...
+    SwkbdType_Latin     = 4,  ///< All Latin like languages keyboard only. (without CJK keyboard)
```
Comment (yellows8) [inline nx/include/switch/applets/swkbd.h]: `All Latin like languages keyboard only (without CJK keyboard).`

File: nx/include/switch/applets/swkbd.h [hunk 6]
```diff
@@ -18,9 +18,15 @@ typedef enum {
...
+    SwkbdType_CHS       = 5,  ///< Chinese Simplified keyboard only.
```
Comment (yellows8) [inline nx/include/switch/applets/swkbd.h]: `SwkbdType_ZhHans`, and `SwkbdType_ZhHant` below, to match LanguageCodes.

File: nx/include/switch/applets/swkbd.h (hunk 5 above)
Comment (Polytoo) [reply nx/include/switch/applets/swkbd.h]: Thank you for your help.

**Pull request merged**